api/envoy/extensions/filters/http/header_to_metadata/v3/header_to_metadata.proto

```diff
@@ -129,4 +129,8 @@ message Config {
 
   // The list of rules to apply to responses.
   repeated Rule response_rules = 2;
+
+  // Configures maximum length for :ref:`header value <envoy_v3_api_field_extensions.filters.http.header_to_metadata.v3.Config.KeyValuePair.value>`
```

Comment:
is the unit for this value Byte? Probably worth mentioning that in the comment.

Reply:
Yes, "header value" is a string. I don't think it's explicitly required to mention.